tc/sgx/trusted_worker_manager/enclave/work_order_data_handler.cpp

```diff
@@ -120,41 +118,46 @@ namespace tcf {
     }
 
     void WorkOrderDataHandler::ComputeHashString(std::string session_key_iv) {
-        ByteArray iv_byte;
+        ByteArray iv_bytes;
         if (iv.empty() && (enc_data_key_str.empty() || "null" == enc_data_key_str)) {
             iv = session_key_iv;
         }
-
-        std::string encrypted_output_str = EncryptOutputData(iv);
+        iv_bytes = HexStringToBinary(iv);
+        std::string encrypted_output_str = EncryptOutputData(iv_bytes);
         hash = tcf::crypto::ComputeMessageHash(workorder_data.decrypted_data);
         Base64EncodedString hash_str = ByteArrayToHexEncodedString(hash);
         concat_string = hash_str + encrypted_output_str + enc_data_key_str + iv;
 
     }
 
-    void WorkOrderDataHandler::VerifyInputHash(ByteArray input_data, ByteArray input_hash) {
+    void WorkOrderDataHandler::VerifyInputHash(ByteArray input_data,
+		                               ByteArray input_hash) {
         // Do nothing at the phase 1
```

Comment:
can we implement it

Reply:
Sure, will do implement and raise another PR